src/Core/Localization/Currency.php

```diff
@@ -166,7 +181,7 @@ public function getNumericIsoCode()
      */
     public function getSymbol($localeCode)
     {
-        if (!array_key_exists($localeCode, $this->symbols)) {
+        if (empty($this->symbols[$localeCode])) {
```

Comment:
What if we create a symbol with '0' :trollface: 
Maybe?
```suggestion
        if (!isset($this->symbols[$localeCode])) {
```

Reply:
Alright